integration-tests/fixtures/generate-bundle-stats/standalone/dotenv-vercel/__snapshots__/standalone-analyzer.test.ts.snap

```diff
@@ -0,0 +1,17 @@
+// Bun Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This is the start of the `integration-tests` for the new `standalone-analyzer` that calls `test-api` at `https://localhost:8000` (run via docker-compose)